etc/docker/docker_entrypoint.sh

```diff
@@ -67,12 +72,14 @@ parse_args "$@"
 
 [ -n "$RADIO_URL" ] || RADIO_URL="spinel+hdlc+uart:///dev/ttyUSB0"
 [ -n "$TUN_INTERFACE_NAME" ] || TUN_INTERFACE_NAME="wpan0"
+[ -n "$BACKBONE_INTERFACE_NAME" ] || BACKBONE_INTERFACE_NAME="eth0"
```

Comment:
remove the default value and only pass `-B` in below Line#92 when specified, thus compatible with previous default behavior.

Reply:
done